libs/ui-components/src/lib/forms/SingleInputForm/SingleInputForm.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import SingleInputForm from "./SingleInputForm";
+
+export const WithDefault = () => {
+
+  const copy = {
```

Comment:
can we import from the copy file

Reply:
We don't want the library to be consuming from the consumer.